thoth/storages/graph/postgres.py

```diff
@@ -3475,6 +3475,17 @@ def update_kebechet_github_installations_on_is_active(self, slug: str) -> bool:
                 return True
             return False
 
+    def get_active_kebechet_github_installations_repos(self) -> Dict[str, List[str]]:
+        """Get all active repositories names with active Kebechet installation."""
```

Comment:
Please enter an example of the method and the returned output. See: https://github.com/thoth-station/storages/blob/196fdfa3ba8c8b0f08ffce0bca434e0d3cc6efe8/thoth/storages/graph/postgres.py#L4138 for example